media-sound/bitwig-studio/bitwig-studio-1.3.16.ebuild

```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+# TODO
+# - Verify all runtime dependencies, min versions, and required flags
+# - Avoid bundled JRE?
+# - use EAPI=6
+
+EAPI=5
+inherit eutils fdo-mime unpacker
+
+DESCRIPTION="for creation and performance of your musical ideas on stage or in the studio."
+HOMEPAGE="http://bitwig.com/"
+SRC_URI="http://downloads.bitwig.com/${P}.deb"
+LICENSE="Bitwig"
+SLOT="1"
+KEYWORDS="~amd64"
+RESTRICT="mirror strip"
```

Comment:
@mrVanDalo is the strip RESTRICT necessary? I'm assuming the binaries are stripped, but you never know.
Also we could make this more specific by using `QA_PRESTRIPPED`. Would that be better/beneficial?